pom.xml

```diff
@@ -98,16 +98,6 @@
           </dependency>
         </dependencies>
         <executions>
-          <execution>
```

Comment:
Should we replace it with an execution with the hibernate style?

Reply:
Yup, this was actually already being done but I renamed the files for clarity. I put all of the Hibernate checks into custom-checks.xml. Now just renamed the xml files to be more clear.